### ISSUE TITLE
Question: Offline support

### ISSUE THREAD

**Issue by daudrain** [pbek/qt-piwik-tracker#4]
Hi,

Does this implementation support offline event tracking?

Thank you,
David

**Comment by pbek** [comment]
No, sorry. Just "real time" by directly sending requests to the Matomo server.
